### ISSUE TITLE
numbering?

### ISSUE THREAD

**Issue by jbenet** [protocol/ipfs-examples#4]
Maybe we should number examples:

```
0-<example>
1-<example>
...
```

so that the `examples/` directory is sorted by creation.

**Comment by RichardLitt** [comment]
I agree with this. Should we rename the folders? Looks like we can use `refresh` and a canonical link to support redirects from the old links, [as shown here](https://stackoverflow.com/questions/10178304/github-jekyll-old-pages-redirection-best-approach).
